tests/MockNativeFunctionTraitTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace RonasIT\Support\Tests;
+
+use RonasIT\Support\Traits\MockNativeFunctionTrait;
+
+class MockNativeFunctionTraitTest extends HelpersTestCase
+{
+    use MockNativeFunctionTrait;
+
+    public function testMockSingleCall()
+    {
+        $this->mockNativeFunction(
+            'RonasIT\Support\Tests',
+            [
+                $this->functionCall('random_bytes', [20], '1234567890')
+            ]
+        );
+
+        $this->assertEquals('1234567890', random_bytes(20));
+    }
+
+    public function testMockSeveralCalls()
+    {
+        $this->mockNativeFunction(
+            'RonasIT\Support\Tests',
+            [
+                $this->functionCall('rand', [0, 9], 5),
+                $this->functionCall('rand', [0, 9], 4),
+                $this->functionCall('rand', [0, 9], 3),
+                $this->functionCall('rand', [0, 9], 2),
+            ]
+        );
+
+        $generate = function(int $length): string {
+            $code = '';
+
+            for ($i = 0; $i < $length; $i++) {
+                $code .= rand(0, 9);
+            }
+
+            return $code;
+        };
+
+        $this->assertEquals('5432', $generate(4));
+    }
+
+    public function testMockSeveralCallsInOneDefinition()
```

Comment:
could you please refine the difference between this and previous test case?

Reply:
Forgot to remove. Fixed